leasing/models/lease.py

```diff
@@ -983,6 +983,7 @@ def determine_payable_rents_and_periods(  # noqa: TODO
                             date_range_start=start_date, date_range_end=end_date
                         ),
                         "last_billing_period": False,
+                        "override_receivable_type": rent.override_receivable_type,
```

Comment:
Should this key be added if override_receivable_type is not available?

Reply:
In `leasing/models/lease.py:1025–1029`, `calculate_invoices` checks if it is defined. If it's None, it will use lease's `service_unit.default_receivable_type_rent`.

Reply:
If the "override_receivable_type" key was added conditionally, we would need to add another if statement to test it and add it, and I don't see a reason to do so. This solution is, in my view, a bit cleaner.

Reply:
I think the key can be here as is without an additional check, even with value None. The property on the model is nullable, and the only usecase in `calculate_invoices` function checks whether the value is falsy or not.

Reply:
Is `"override_receivable_type": rent.override_receivable_type | None` a good idea?

Reply:
I don't think it's needed, because the value of `rent.override_receivable_type` should already be None if it is not set to some other value. I think it is fine as is.